src/pages/App/index.testtofix.tsx

```diff
@@ -6,6 +6,8 @@ import { BrowserRouter } from 'react-router-dom';
 import sinon from 'sinon';
 import { store } from '../../util/testUtils';
 
+// FIXME transform of pretty-bytes (jest config?)
```

Comment:
looks like we need to tell jest to transform this dep (probably need to tweak jest config with `transformIgnorePatterns` or `transform` options)